pandas/tests/io/parser/test_parsers.py

```diff
@@ -25,6 +28,18 @@
 from .dtypes import DtypeTests
 
 
+@pytest.mark.high_memory
+def test_bytes_exceed_2gb():
+    """Read from a "CSV" that has a column larger than 2GB.
+
+    GH 16798
+    """
+    csv = StringIO('strings\n' + '\n'.join(
+        ['x' * (1 << 20) for _ in range(2100)]))
+    df = read_csv(csv, low_memory=False)
+    assert not df.empty
```

Comment:
Thanks. Would this be mergeable, then? I've addressed @gfyoung 's comments which I believe were the last outstanding.